lib/server.coffee

```diff
@@ -5,6 +5,18 @@ crypto = Npm.require 'crypto'
 
 cacheCollection = new Mongo.Collection 'SpiderableCacheCollection'
 
+hashCode = (string) ->
```

Comment:
I guess, my point is we don't know without test code. Most likely it has low chance of collision, but we don't have any validation of that without test code and analysis.  Anyone reading this would wonder why we don't use standard libraries, etc. (Why roll our own...)